infra/data-pipeline.yml

```diff
@@ -1210,6 +1210,48 @@ tasks:
     #        owner: release-mgmt-analysis@mozilla.com
     #        source: ${repository}/raw/master/data-pipeline.yml
 
+    - ID: train-needsdiagnosis
```

Comment:
Can you add the task to the dependencies of the `check_metrics` task (https://github.com/mozilla/bugbug/blob/fd8bddb0e1d0a4e8a298105d34158d2940d33f3c/infra/data-pipeline.yml#L1255)?

The `check_metrics` task will download the metrics.json file from the task and generate a chart to show the evolution of the metrics (accuracy, precision and recall) over time, so you can see if there are regressions.